Makefile

```diff
@@ -55,10 +57,9 @@ ccoverage:  ## Run the test suite, with C++ code coverage
 	CFLAGS="$(CFLAGS) -O0 -pg --coverage" $(MAKE) build
 	$(MAKE) check
 	gcov -i build/*/src/pystack/_pystack -i -d
-	lcov --capture --directory .  --output-file pystack.info
-	lcov --extract pystack.info '*/src/pystack/*' --output-file pystack.info
-	genhtml pystack.info --output-directory pystack-coverage
-	find . | grep -E '(\.gcda|\.gcno|\.gcov\.json\.gz)' | xargs rm -rf
+	lcov --capture --directory .  --output-file cppcoverage.lcov
+	lcov --extract cppcoverage.lcov '*/src/pystack/_pystack/*' --output-file cppcoverage.lcov
+	genhtml *coverage.lcov --branch-coverage --output-directory pystack-coverage
```

Comment:
This `*coverage.lcov` means that if there is already a `pycoverage.lcov` in the cwd, it will merge the results of the two in the html report.